### DISCUSSION TITLE
DualCLIPLoaderGGUF 'NoneType' object has no attribute 'endswith'

### DISCUSSION THREAD

**Discussion by Melmahovskye** [comfyanonymous/ComfyUI#5851] [Q&A]
Please help, I dont understand  this error
![image](https://github.com/user-attachments/assets/61061da6-74e0-41f7-a3f1-85237ab2416e)
![image](https://github.com/user-attachments/assets/24363305-160b-4ebc-a4e0-09633751670d)
![image](https://github.com/user-attachments/assets/841e9292-6794-4a62-bfd1-7ab68225124d)

**Comment by LukeG89** [comment]
You should ask for help here: https://github.com/city96/ComfyUI-GGUF

My guess is that the models you are using are not GGUF, try loading them with the normal DualCLIPLoader
